pkg/util/common.go

```diff
@@ -200,6 +201,15 @@ func CreateHTTPTransport() *http.Transport {
 
 	transport := &http.Transport{
 		TLSClientConfig: tlsConfig,
+		DialContext: (&net.Dialer{
+			Timeout:   30 * time.Second,
+			KeepAlive: 30 * time.Second,
```

Comment:
let's set `DualStack` to its default of `true` from the `DefaultTransport` as well?

Reply:
I was a bit afraid as it will probably have effect on the amount of agent that hit our intake using ipv6.

Reply:
Seems like it might get enabled by default in 1.12 
https://github.com/golang/go/issues/22225#issuecomment-426746554

Reply:
I tend to think it's rather safe to enable it, if I understand correctly at worst we'd initially attempt to create 2 connections instead of one. In any case I think that's better than Agent5 which would re-create new connections for every requests (and might've been using this dual-stack logic as well? Not sure).

Let's confirm with metrics-intake that this is safe though.

Reply:
Right now ~1% of the request made to our intake are transported over ipv6. This is likely due to go selecting the pure go resolver which from what I understand (information is very scarce on this) prefers ipv4 over ipv6. If this gives the priority to ipv6 we might have issues. 

I am waiting for an answer from our backend team but I think we should leave this option to `false` for 6.6

Reply:
Okay let's enable it for now. We'll revert if we get an answer that it's not OK